CHANGELOG.md

```diff
@@ -2658,7 +2658,7 @@ Other minor changes
                 ∀ {m n} → _Respectsʳ_ (_<_ {m} {n}) _≋_
   <-wellFounded : Symmetric _≈_ →  Transitive _≈_ → _≺_ Respectsʳ _≈_ → WellFounded _≺_ →
                   ∀ {n} → WellFounded (_<_ {n})
-```
+  ```
```

Comment:
This is a drive-by fix, but important because the formatting of the entire rest of the file is broken without it.

Reply:
Is this fixed now?

Reply:
If it's not showing up as a conflict then it's probably not...

Reply:
Should it be lifted out as a separate 'PR' then (or directly edited on `master`)?

Reply:
Fixed in #2154